product_pack/views/product_pack_line_views.xml

```diff
@@ -14,14 +14,14 @@
             </form>
         </field>
     </record>
-    <record id="product_pack_line_tree" model="ir.ui.view">
-        <field name="name">product.pack.line.tree</field>
+    <record id="product_pack_line_list" model="ir.ui.view">
```

Comment:
Please don't rename XML-IDs for no gain in migrations. It only provokes possible headaches.